x-pack/filebeat/input/s3/s3_integration_test.go

```diff
@@ -259,46 +247,32 @@ func TestS3Input(t *testing.T) {
 
 		svcS3 := s3.New(awsConfig)
 		uploadSampleLogFile(t, s3BucketNameEnv, svcS3)
+		t.Log("sleeping 30 seconds for uploading sample log file")
 		time.Sleep(30 * time.Second)
+		t.Log("done sleeping")
```

Comment:
That is a long sleep and a potential source for flakiness. Can we somehow get a signal that the upload is finished or continuously poll for the state?

Reply:
This is also moved for now because we will have a separate PR adding terraform into s3 input integration test and also enabling cloud tests for filebeat in CI.